testapps/iOSImageFormatProfilingTestApp/test/ViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+import UIKit
+
+class ResettableView: UIView {
+	override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+		super.touchesEnded(touches, with: event)
+		subviews.forEach { $0.removeFromSuperview() }
+		setNeedsLayout()
+		setNeedsDisplay()
+		layoutIfNeeded()
+	}
+}
+
+class ViewController: UIViewController {
+
+	var imagesContainerView = ResettableView()
+	let stretchingSwitch = UISwitch()
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
```

Comment:
minor: should probably be a loadView. I think we should have a guideline on loadView/viewDidLoad so would be good for code in our repo to conform to all our guidelines